### PR TITLE
Mergeback 1.5.0 to develop

### PR DESCRIPTION
**Pull request by yunchu**
### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

File: tests/assets/classification_dataset_class_incremental/3/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```
Comment (sungmanc) [inline tests/assets/classification_dataset_class_incremental/3/.gitignore:1]: Is this file intended?

File: docs/source/guide/explanation/algorithms/object_detection/object_detection.rst [hunk 2]
```diff
@@ -117,6 +117,26 @@ In addition to these models, we supports experimental models for object detectio
 `Deformable_DETR <https://arxiv.org/abs/2010.04159>`_ is `DETR <https://arxiv.org/abs/2005.12872>`_ based model, and it solves slow convergence problem of DETR. `DINO <https://arxiv.org/abs/2203.03605>`_ improves Deformable DETR based methods via denoising anchor boxes. Current SOTA models for object detection are based on DINO.
 `Lite-DINO <https://arxiv.org/abs/2303.07335>`_ is efficient structure for DINO. It reduces FLOPS of transformer's encoder which takes the highest computational costs.
 
+.. note::
```
Comment (sungmanc) [inline docs/source/guide/explanation/algorithms/object_detection/object_detection.rst:120]: Is this related to the 1.5.0? Or missed from previous merge-back?

**Review by sungmanc — DISMISSED** [review]
LGTM, I left 2 minor questions

File: README.md
Comment (goodsong81) [inline README.md:1]: stable -> latest

File: docs/source/guide/release_notes/index.rst
Comment (goodsong81) [inline docs/source/guide/release_notes/index.rst:1]: Would you update 1.5.0 part?

File: tests/assets/classification_dataset_class_incremental/3/.gitignore (hunk 1 above)
Comment (goodsong81) [reply tests/assets/classification_dataset_class_incremental/3/.gitignore:1]: Yes, from https://github.com/openvinotoolkit/training_extensions/pull/2606 to simulate a label without annotation.

**Review by goodsong81 — CHANGES_REQUESTED** [review]
Looks OK to me. Could you fix a few minor things?

File: docs/source/guide/explanation/algorithms/object_detection/object_detection.rst (hunk 2 above)
Comment (yunchu) [reply docs/source/guide/explanation/algorithms/object_detection/object_detection.rst:120]: i think it was added at 1.5.0. I've checked here https://github.com/openvinotoolkit/training_extensions/compare/develop...releases/1.5.0

File: docs/source/guide/release_notes/index.rst
Comment (yunchu) [reply docs/source/guide/release_notes/index.rst:1]: I think release note on the doc should be updated on the releases/1.5.0. after that I need to merge back once again. thanks for your comment.

File: README.md
Comment (yunchu) [reply README.md:1]: do you think all links in the develop should be consisted with 'latest'?

Comment (goodsong81) [reply README.md:1]: I have no much preference, but it would be better to be consistent at least.
I thought it was by mistake. If it's intentional, please go ahead.

**Review by goodsong81 — DISMISSED** [review]
Thank you!

File: README.md
Comment (yunchu) [reply README.md:1]: actually my intention was that at least, readme.md should use 'latest' internal docs link to represent the latest updates for the `develop`. so, i think i have to update some part for that.

**Review by goodsong81 — APPROVED** [review]

**Review by chuneuny-emily — APPROVED** [review]
